ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -128,12 +129,16 @@ export const useNewMhrRegistration = () => {
       // @ts-ignore - TODO: Mhr-Submission - api asks for number, maybe fix this once step 3 is finished?
       ownerGroup.groupId = parseInt(ownerGroup.groupId)
 
-      ownerGroup.type = 'SO'// TODO: Mhr-Submission - DELETE after step 3 has been completed
+      ownerGroup.type = getTenancyTypeFromDescription(ownerGroup.type)
```

Comment:
Hmm, could be much simpler: `ownerGroup.type = HomeTenancyTypes[ownerGroup.type]`

Reply:
At this point ownerGroup.type is equal to the value of the enum.  eg. 'Sole ownership','Tenants in Common','Joint Tenants"

A conversion is required to use the 'key' value of the enum for the db. 
`export enum HomeTenancyTypes {
  JOINT = 'Joint Tenants',
  SOLE = 'Sole Ownership',
  COMMON = 'Tenants In Common',
  NA = 'N/A'
}`

Reply:
Ohh, you are trying to achieve the opposite..

Reply:
Then maybe we can do: `Object.keys(HomeTenancyTypes).find(key => HomeTenancyTypes[key] === ownerGroup.type)`

Reply:
Ill give it a try.. looks cleaner.

Reply:
Change made.